charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -141,21 +141,75 @@ data:
                 - name: OT_ENABLED
                   value: {{ .Values.application.ray.openTelemetryCollector.enabled | quote }}
 {{- end }}
-            - image: fluent/fluent-bit:1.9.10
-              name: ray-head-logs
+            - image: {{ .Values.application.ray.proxyImage }}
+              securityContext:
+                runAsGroup: 123
+                privileged: true
+                capabilities:
+                  add: ["NET_ADMIN"]
+              imagePullPolicy: IfNotPresent
+              name: proxy
               resources:
                 limits:
-                  cpu: 100m
-                  memory: 128Mi
+                  cpu: {{ .Values.application.ray.cpu }}
+                  memory: {{ .Values.application.ray.memory }}Gi
```

Comment:
does the proxy need the same amount of resources as the ray container(s)? if so that would essentially double our compute requirements

Reply:
Probably not.  I'll add separate variables